android/routes/ui/src/main/java/com/trafi/routes/ui/RoutesScreen.kt

```diff
@@ -198,6 +207,39 @@ private fun ColumnScope.LocationSearchBody(
     }
 }
 
+@Composable
+fun RouteSearchTabsList(
+    modifier: Modifier,
+    vehicleTypes: List<TabItem>,
+    selectedItemId: String,
+    onVehicleClick: (String) -> Unit,
+    itemModifier: Modifier = Modifier,
```

Comment:
Will we need to customize items from the outside? Seems like this parameter could be omitted.